tests/whitelist

```diff
@@ -0,0 +1,26 @@
+# Add lines to this file to list tests to skip
+
+# Test | Resource
+# Blocked Restriction Test 1
+t202-elemrestriction /cpu/occa
+t202-elemrestriction /gpu/occa
+t202-elemrestriction /omp/occa
+t202-elemrestriction /ocl/occa
+t202-elemrestriction /gpu/magma
+t202-elemrestriction-f /cpu/occa
+t202-elemrestriction-f /gpu/occa
+t202-elemrestriction-f /omp/occa
+t202-elemrestriction-f /ocl/occa
+t202-elemrestriction-f /gpu/magma
```

Comment:
I am listing the Fortran and C tests separately. I did it this way because we have had trouble with the Fortran tests not being the same as the C tests and certain backends not playing nice over Fortran.